src/site/content/en/blog/reporting-api-migration/index.md

```diff
@@ -0,0 +1,471 @@
+---
+layout: post
+title: Migrate to the Reporting API v1
+subhead: |
+  A new version of the Reporting API is available. It's simpler, more private, and more likely to be supported across browsers.
+authors:
+  - maudn
+date: 2021-06-14
+updated: 2021-06-14
+description: |
+  Migrate to the new Reporting API.
+hero: image/O2RNUyVSLubjvENAT3e7JSdqSOx1/PYEe5UP3bVYzPMXdQc0X.jpg
+alt: |
+  A person and their refection on wet sand, symbolizing the migration from the legacy to the new Reporting API.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - monitoring
+  - security
+---
+
+{% Banner 'caution', 'body' %} This post covers how to migrate to the new version of the
+Reporting API (v1). If you're new to the Reporting API or need detailed usage examples,
+head over to the [main post on the Reporting API](/reporting-api) instead. {% endBanner %}
+
+The Reporting API informs you about errors that happen across your site as visitors use
+it. It gives you visibility on browser interventions, browser crashes,
+Content-Security-Policy violations, COOP/COEP violations, deprecation warnings and more.
+
+**A new version of the Reporting API is available. It's simpler, more private, and more
+likely to be supported across browsers.**
+
+{% Aside %} The legacy Reporting API is named _Reporting API v0_. The new Reporting API is
+named _Reporting API v1_. {% endAside %}
+
+## Summary
+
+{% Banner 'info', 'body' %}
+
+If you already have reporting functionality for your site, **migrate to v1**
+(`Reporting-Endpoints` header) but keep the v0 header (`Report-To`) around for some time.
+
+If you're adding reporting functionality to your site just now, use only v1 (`Reporting-Endpoints` header).
+
+Make sure to set the `Reporting-Endpoints` header on all responses that might generate
+reports.
+
+If you're maintaining an endpoint service or are operating your own, expect **more
+traffic** as you or external developers migrate to v1 (`Reporting-Endpoints` header).
+
+{% endBanner %}
+
+Keep reading for details and example code!
+
+## Network Error Logging
+
+{% Banner 'caution', 'body' %}
+
+If you use [Network Error Logging](https://w3c.github.io/network-error-logging/), continue using `Report-To` (v0) because Network Error Logging isn't
+supported in the Reporting API v1. A new mechanism for Network Error Logging will be
+developed⏤until that's available, the Reporting API v0 should be used.
+**As of June 2021**, this new mechanism is not shipped yet.
+
+{% endBanner %}
+
+## Demo and code
+
+- [Demo site: new reporting API (v1)](https://reporting-api-demo.glitch.me)
```

Comment:
I'm on Chrome 91.0.4472.77 and at first I got 0 reports on the reporting endpoint page. Tried refreshing multiple times. I

When I tried it in Canary, I initially got 1 report, then it jumped to 4 reports. I had that same page opened in my Chrome 91 instance and it also got updated to show 4 reports.

I also couldn't find the --short-reporting-delay flag in either 91 or Canary.

AI:
- Verify that this really works in Chrome 91.
- Check the demo behaviour when multiple users visit it.
- Clarify where to find the flag and what it does.

Another thing: You might want to "boost" this app on Glitch, it was asleep when I clicked and it took a while for it to load.

Reply:
Once that's cleared up, consider embedding this demo. I know there's not much to see on that first demo page, but at least readers will get a preview of what the demo is about.

Reply:
> When I tried it in Canary, I initially got 1 report, then it jumped to 4 reports
I think this is due to the missing flag --short-reporting-delay.

>  I also couldn't find the --short-reporting-delay flag in either 91 or Canary.
Oh so it's a CLI flag only, not available from the UI! Will need to update the text so readers aren't confused.

Can you maybe try the demo with the flag on (via your terminal)? Instructions are in the article.
Thank you!

Reply:
The correct version number was 93. Done:
- Updated version number to 93 in the demo
- Clarified that the flags are CLI-only i.e. not in the Chrome UI, in both the demo page and here in the article where relevant https://deploy-preview-5580--web-dev-staging.netlify.app/reporting-api/#saving-time

> Another thing: You might want to "boost" this app on Glitch, it was asleep when I clicked and it took a while for it to load.
oh yes great tip. I thought that glitch demos under the web.dev team were automatically boosted. Is it not the cae and so I need to boost it manually?

Reply:
Demos are now boosted.
(Note: TIL glitch demos under the web.dev team are automatically rate-UNlimited, but not automatically boosted. I purchased a Pro membership in order to boost them.)